troposphere/batch.py

```diff
@@ -10,13 +10,13 @@ class ComputeResources(AWSProperty):
         "SecurityGroupIds": ([basestring], True),
         "BidPercentage": (positive_integer, False),
         "Type": (basestring, True),
-        "Subnets": ([basestring], True),
+        "Subnets": (list, True),
```

Comment:
Any reason why this was changed? Using [basestring] is a list of basestrings which is more specific than just using list.

Reply:
In a previous version of this PR, the example template used a `CommaDelimitedList` for the `Subnets` parameter type and the build failed for Python 2.7 (see https://travis-ci.org/cloudtools/troposphere/jobs/292160894). Not sure why it failed though…

So I figured I'd use the `list` property type as in `AWS::RDS::DBSubnetGroup.SubnetIds` or `AWS::ElasticLoadBalancing::LoadBalancer.Subnets`.